src/ffi/client.rs

```diff
@@ -94,7 +94,6 @@ ffi_fn! {
     /// Creates a new set of HTTP clientconn options to be used in a handshake.
     fn hyper_clientconn_options_new() -> *mut hyper_clientconn_options {
         let mut builder = conn::Builder::new();
-        builder.http1_preserve_header_case(true);
```

Comment:
Looks like this removal makes a lint trigger about no longer needing to be `let mut builder`.